app/models/customer.rb

```diff
@@ -52,4 +53,9 @@ def check_for_orders
     errors.add(:base, I18n.t('admin.customers.destroy.has_associated_orders'))
     throw :abort
   end
+
+  def set_unused_address_fields
+    ship_address.company = 'Company' if ship_address.present?
+    bill_address.company = 'Company' if bill_address.present?
```

Comment:
`ship_address` and `bill_address` are either objects or `nill`, right? So guess that this should work?
```suggestion
    ship_address&.company = 'Company'
    bill_address&.company = 'Company'
```

Reply:
Just wondering: other code below uses `unused` to fill required but unused address attributes. Why aren't you doing the same here?

Reply:
@mkllnk It's because in `services/order_factory.rb` its set as 'Company'. So if we want the specs to pass, they have to be set to 'Company'.

Reply:
That is just example data that can be changed however we like. Before your PR, that field was completely unused. So feel free to change the factory:
https://github.com/openfoodfoundation/openfoodnetwork/blob/02a10ede152a21f7bac8dcc386b102903a2b5aa5/spec/factories/address_factory.rb#L7